### PR TITLE
deploy/docker: Update pocli version

### LINKED ISSUE #268: BUG: Pocli version used in docker deployment is not the latest version
### What version of the package are you using?
v0.0.2410

### Does this issue reproduce with the latest release?
Yes

### What OS and CPU architecture are you using (`go env`)?
This does not relate to the issue: using Docker deployment.

### What did you do?
Used Docker deployment.

```
git clone https://github.com/veraison/services.git
cd services/deployments/docker
make
source env.bash
veraison start
```
Tried (based on installed `pocli` version):
```
pocli create CCA_SSD_PLATFORM accept-all.rego
```
and (based on latest `pocli` version):
```
pocli create ARM_CCA accept-all.rego -i
```

### What did you expect to see?
Second `pocli` call succeeds.

Docker deployment uses `pocli@2fa24ea3` version, the same the native deployment uses
[deployments/native/deployment.sh#L216](https://github.com/veraison/services/blob/v0.0.2410/deployments/native/deployment.sh#L216):
```
	_init_client pocli github.com/veraison/pocli@2fa24ea3
```

### What did you see instead?
Call to:
```
pocli create CCA_SSD_PLATFORM accept-all.rego
```
fails, because `pocli` tries to connect using http instead of expected https.

Call to:
```
pocli create ARM_CCA accept-all.rego -i
```
fails, because `pocli` does not recognize `-i` option.

Docker deployment uses `latest` version, which is version v0.1.0 from September of 2023.
[deployments/docker/src/builder.docker#L65:](https://github.com/veraison/services/blob/v0.0.2410/deployments/docker/src/builder.docker#L65)
```
    go install github.com/veraison/pocli@latest &&\
```

### PR DESCRIPTION
**Pull request by zos**
Upgrade the pocli tool in the docker deployment to the latest version as it is in native deployment.

Fix #268

### CODE REVIEW

**Comment by thomas-fossati** [comment]
Thanks @zos ! I have just added a [v0.2.0](https://github.com/veraison/pocli/releases/tag/v0.2.0) tag to `veraison/pocli`.  You could use that in this change.

File: deployments/docker/src/builder.docker [hunk 1]
```diff
@@ -62,7 +62,7 @@ RUN go mod download &&\
     go install github.com/mitchellh/protoc-gen-go-json@v1.1.0 &&\
     go install github.com/veraison/corim/cocli@be7ec482 &&\
     go install github.com/veraison/evcli/v2@86d12893 &&\
-    go install github.com/veraison/pocli@latest &&\
+    go install github.com/veraison/pocli@2fa24ea3 &&\
```
Comment (thomas-fossati) [inline deployments/docker/src/builder.docker:65]:
```suggestion
    go install github.com/veraison/pocli@v0.2.0 &&\
```

**Review by thomas-fossati — APPROVED** [review]
🥳 welcome @zos !

**Comment by zos** [comment]
Thank you :)!

**Pull request merged**